mvp-spec/README.md

```diff
@@ -34,8 +34,8 @@ The operator is responsible for:
 
 Key features of PAF include:
 
-- signing transmissions and request with private keys
-- making available alongside the ad an audit log of entities involved
+- signing communication, transmissions and request with private keys
```

Comment:
request**s**?

Reply:
@OlivierChirouze 
To get it right, what is being signed is:
-  The Seed
- Transmissions Requests and Transmissions Responses that are actually payloads embedded with bid requests and bid responses

Anything else? Thanks

Reply:
yep!
- ids
- preferences
- all messages sent to or from the operator

Reply:
Thanks, commit added